### ISSUE TITLE
Check number of volumes attached to a droplet

### ISSUE THREAD

**Issue by fatih** [digitalocean/csi-digitalocean#3]
Related: https://github.com/container-storage-interface/spec/issues/217

**Comment by zarbis** [comment]
```
Support for dynamic maximum volume count has been introduced as an alpha feature.
```
https://github.com/kubernetes/features/issues/554

**Comment by fatih** [comment]
This is postponed until K8S v1.12 is out as the CSI integration is not included in v1.11

**Comment by fatih** [comment]
This is now implemented via Spec `v0.3.0` and is merged to master: https://github.com/digitalocean/csi-digitalocean/blob/master/driver/node.go#L300 (was part of #72 ). I'll keep it open though to see if it indeed works in `v1.12`.

**Comment by timoreimann** [comment]
This one has most likely proven to be working fine by now.
